From d3033163a11c32370fb3977b0c5391ba144885dc Mon Sep 17 00:00:00 2001
From: Martin Weinelt <martin@darmstadt.freifunk.net>
Date: Mon, 3 Jun 2019 00:12:19 +0200
Subject: [PATCH] modules: update routing

c07326c batman-adv: Fix duplicated OGMs on NETDEV_UP

fixes #1446

(cherry picked from commit 9e00ecd8e42f3466566b8dd37444e1f02299d05d)
---
 modules | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/modules b/modules
index 9ab1b848..756db33e 100644
--- a/modules
+++ b/modules
@@ -10,7 +10,7 @@ PACKAGES_PACKAGES_COMMIT=5d3cb594e49132158ca7b41f4246421078e8f92a
 
 PACKAGES_ROUTING_REPO=https://github.com/openwrt-routing/packages.git
 PACKAGES_ROUTING_BRANCH=openwrt-18.06
-PACKAGES_ROUTING_COMMIT=cad1fba86fa5fa787332d27e371a461afec0ad32
+PACKAGES_ROUTING_COMMIT=7589804a56baac804421b492c93004c28a627abb
 
 PACKAGES_LUCI_REPO=https://github.com/openwrt/luci.git
 PACKAGES_LUCI_BRANCH=openwrt-18.06
-- 
2.17.1

